modules/cas_cache/layer_cache_management.c

```diff
@@ -777,6 +777,12 @@ int cache_mngt_purge_object(const char *cache_name, size_t cache_name_len,
 	if (result)
 		return result;
 
+	if (ocf_cache_is_passive(cache))
+	{
+		ocf_mngt_cache_put(cache);
+		return OCF_ERR_CACHE_PASSIVE;
```

Comment:
If this error always originates in CAS Linux kernel adapter, you can define it in the same module with "KCAS_ERR_" prefix